Rules/CommonScripts/KillMessages.as

```diff
@@ -276,7 +280,7 @@ void onPlayerDie(CRules@ this, CPlayer@ victim, CPlayer@ killer, u8 customdata)
 			CPlayer@ helper = getAssistPlayer(victim, killer);
 
 			bool kill = killerblob !is null && victimblob !is null && killerblob.isMyPlayer() && killerblob !is victimblob;
-			bool assist = helper !is null;
+			bool assist = helper !is null && helper.isMyPlayer();
```

Comment:
What is this change for?

Reply:
Even in the base script (https://github.com/transhumandesign/kag-base/blob/master/Rules/CommonScripts/KillMessages.as) it is like that; so it's not actually a change nor is it an addition.
Apparently, I changed it before the assist kill message was fixed (the issue when the +1 message appeared for everyone), since I was working as well on getting the assists fixed back then. :thinking:

Reply:
Eeeh. Seems like a GitHub bug to me.